README.md

```diff
@@ -324,9 +326,11 @@ To use this binaries and libraries, you should update the necessary environment
 
 Set the environment variable `ROBOTOLOGY_SUPERBUILD_ROOT` so that it points to the  directory where you cloned the robotology-superbuild repository.
 
-Append `$ROBOTOLOGY_SUPERBUILD_ROOT/build/install/bin` to your PATH.
+Append `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/bin` to your PATH.
```

Comment:
I am a bit confused by all this changes to Windows section of the README, given that `DYLD_LIBRARY_PATH` does not exist on Windows. Did you intended to do those in the macOS section?

Reply:
Wrong Ctrl+F. My fault, thanks for catching it

Reply:
This README is huge, it's too easy to get lost in the sections :disappointed_relieved: